src/compiler.js

```diff
@@ -63,7 +63,7 @@ module.exports.run = (faviconOptions, context, compilation) => {
       // Replace [hash] placeholders in filename
       const result = extractAssetFromCompilation(
         compilation,
-        compilation.mainTemplate.getAssetPath(filename, { hash, chunk })
```

Comment:
There is one more place which needs tobe changed https://github.com/jantimon/favicons-webpack-plugin/blob/ed6c0217e349675d09ab456139b96918ef6d727a/src/index.js#L182

_inb4 I'm not maintainer, just researching before made my own PR 🤷🏻‍♂️_